docs/source/user/settings.rst

```diff
@@ -308,6 +308,29 @@ Specifies the datetime format of OpenVPN management status parser used by the
 :ref:`convert_called_station_id <convert_called_station_id>`
 command.
 
+``OPENWISP_RADIUS_UNVERIFY_INACTIVE_USERS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``0`` (disabled)
+
+Number of days from user's ``last_login`` after which the
+user will be flagged as *unverified*.
+
+When set to ``0``, the feature would be disabled and the user will
+not be flagged as *unverified*.
+
+``OPENWISP_RADIUS_DELETE_INACTIVE_USERS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``540`` (18 months)
```

Comment:
The default here looks incorrect to me.

Reply:
Are you referencing the default value? If so, the issue asked for the default value of 18 months.

Reply:
The values in the README and code do not match, let's set this to disabled by default in both.